From fc18bce487ed0fcf2d46047bed1eebad0ee869b9 Mon Sep 17 00:00:00 2001
Message-ID: <fc18bce487ed0fcf2d46047bed1eebad0ee869b9.1699403993.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:56:31 +0000
Subject: [PATCH 58/67] gnu: go-github-com-xeipuuv-gojsonpointer: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-xeipuuv-gojsonpointer): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I1c77a5a1b0e85fa57578ff29c9e3c403e4d62bc8
---
 gnu/packages/golang-web.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm     | 25 -------------------------
 2 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 346ca87cc7..1f9fe0ffca 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -1202,6 +1202,31 @@ (define-public go-github-com-valyala-fasthttp
 replacement for native @code{net/http} module.")
     (license license:expat)))
 
+(define-public go-github-com-xeipuuv-gojsonpointer
+  (let ((commit "4e3ac2762d5f479393488629ee9370b50873b3a6")
+        (revision "0"))
+    (package
+      (name "go-github-com-xeipuuv-gojsonpointer")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/xeipuuv/gojsonpointer")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "13y6iq2nzf9z4ls66bfgnnamj2m3438absmbpqry64bpwjfbsi9q"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/xeipuuv/gojsonpointer"))
+      (home-page "https://github.com/xeipuuv/gojsonpointer")
+      (synopsis "Implementation of JSON Pointer for Go")
+      (description
+       "This package provides an implementation of JSON Pointer for the Go
+programming language.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-xeipuuv-gojsonschema
   (let ((commit "6b67b3fab74d992bd07f72550006ab2c6907c416")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 16b13cd59e..fb01dd4679 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -12125,31 +12125,6 @@ (define-public go-github-com-go-test-deep
 when comparing complex types like structures and maps.")
     (license license:expat)))
 
-(define-public go-github-com-xeipuuv-gojsonpointer
-  (let ((commit "4e3ac2762d5f479393488629ee9370b50873b3a6")
-        (revision "0"))
-    (package
-      (name "go-github-com-xeipuuv-gojsonpointer")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/xeipuuv/gojsonpointer")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "13y6iq2nzf9z4ls66bfgnnamj2m3438absmbpqry64bpwjfbsi9q"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/xeipuuv/gojsonpointer"))
-      (home-page "https://github.com/xeipuuv/gojsonpointer")
-      (synopsis "Implementation of JSON Pointer for Go")
-      (description
-       "This package provides an implementation of JSON Pointer for the Go
-programming language.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-xeipuuv-gojsonreference
   (let ((commit "bd5ef7bd5415a7ac448318e64f11a24cd21e594b")
         (revision "0"))
-- 
2.41.0

